server/src/main/java/org/elasticsearch/repositories/RepositoriesModule.java

```diff
@@ -36,6 +36,7 @@
 public final class RepositoriesModule {
 
     public static final String METRIC_REQUESTS_COUNT = "repositories.requests.count";
+    public static final String HTTP_REQUEST_TIME_IN_MICROS_COUNT = "repositories.requests.httpRequestTimeInMicros.count";
```

Comment:
In the "Metrics Naming Guideline Proposal" doc it says not to include units in the metric name but instead put them in the unit and to use snake_case instead of camelCase. So I would rename the metric to something like:
`repositories.requests.http_request_time.count` and keep the `micros` as unit.